### DISCUSSION TITLE
Is there a way to use asyncpg's "copy_from_table" or psycopg2's "copy_from" in databases?

### DISCUSSION THREAD

**Discussion by ohmeow** [encode/databases#305] [Q&A] [answered]
I'm doing big bulk inserts into postgres, but can' figure out if/how databases exposes these methods?  If its possible, please give me a hollar.

Thanks - wg

**Comment by elivin** [comment, marked as answer]
Hi

I found one way to do it. There is a `raw_connection` property that returns the driver's current connection. It can be used like this:
```python
db = Database("postgresql+asyncpg://...")
# ...

async with db.connection() as connection:
    await connection.raw_connection.copy_records_to_table(
        "table_name",
        columns=["id", "name"],
        records=[
            (1, "Record 1"),
            (2, "Record 2"),
            (3, "Record 3"),
        ],
    )
```

In this sample `connection.raw_connection` is an [asyncpg.Connection object](https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg.connection.Connection.copy_records_to_table), so we can use all of its methods.
